pkg/server/oauth_github.go

```diff
@@ -11,18 +11,18 @@ import (
 	"golang.org/x/oauth2"
 )
 
-type oauthHanlderGithub struct {
+type oauthHandlerGithub struct {
```

Comment:
this is funny, I looked at them so many times and never noticed

Reply:
me too, it was only once I installed VSCode extension for spelling errors that I noticed